server/face-recognitor/face-recognitor.js

```diff
@@ -0,0 +1,59 @@
+const fs = require('fs');
+const faceRecognition = require('face-recognition');
+const path = require('path');
+
+const fr = faceRecognition
+const detector = fr.FaceDetector()
+
+// Loads a presaved model from ./model.json
+const modelPath = process.env.NODE_ENV === 'production' ?
+  'model.json' :
+  './server/face-recognitor/model.json'
+
+const model = JSON.parse(fs.readFileSync(modelPath))
+
+const recognizer = fr.FaceRecognizer()
+recognizer.load(model)
+
+// size of the detected faces in pixels (rectangles)
+const size = 150
+
+// Predicts the players in a picture
+const whoIsIt = imagePath => {
+
+  // Load image from path
+  const image = fr.loadImage(path.resolve(imagePath))
+
+  const rects = detector.locateFaces(image, size).map(mrect => mrect.rect)
+  if ( !rects.length ) return []
+
+  const faces = detector.getFacesFromLocations(image, rects, size)
+  if ( !faces.length ) return []
```

Comment:
I'm fairly sure this check is redundant, since the next faces.map...(etc) will returns [] anyway if faces.length is 0

Also, `{ return [] }` but I'll leave that up to you.